scripts/convert-svgs-to-icon-mixins.js

```diff
@@ -31,10 +31,20 @@ function printBaseMixins(svgs) {
   console.log('//------------------------');
 
   svgs.forEach((svg) => {
-    console.log(`@mixin vf-icon-${svg.name}(${svg.colors.join(', ')}) {`);
-    console.log(` background-image: url("data:image/svg+xml,${encodeSVG(svg.svg, svg.colors)}");`);
-    console.log('}');
-    console.log('');
+    console.log(`
+// ${svg.name}
+@function vf-icon-${svg.name}-url($color) {
+  @return url("data:image/svg+xml,${encodeSVG(svg.svg, svg.colors)}");
+}
+
+@mixin vf-icon-${svg.name}($color: $colors--light-theme--icon) {
+  background-image: vf-icon-${svg.name}-url($color);
+}
+
+@mixin vf-icon-${svg.name}-themed {
+  @include vf-themed-icon($light-value: vf-icon-${svg.name}-url($colors--light-theme--icon), $dark-value: vf-icon-${svg.name}-url($colors--dark-theme--icon));
+}
+`);
```

Comment:
Are these lines (and the lines logged out by the `console.log` in `svgs.forEach` intentionally un-indented for ease of use when using the conversion script output?

Maybe it would be better to properly indent / format them here, and people can use the repo's formatting tools on the output before committing if they need to.

Reply:
Yes, this is to keep the output unintended so it can be directly used. But it's a fair suggestion that formatting will handle it. 

I'll see how bad the output would be if the code is properly intended here.

Reply:
Fixed.